src/components/TalentsListPage/TalentsListPage.jsx

```diff
@@ -1,14 +1,17 @@
 import React from "react";
 import { Layout } from "../Layout";
-import { SearchPanel } from "../TalentsListPage/SearchPanel/SearchPanel";
+import { SearchPanel } from "./SearchPanel";
 import { Filter } from "./Filter";
 import { TalentsList } from "./TalentsList/TalentsList";
```

Comment:
you didnt change import TalentList component. When index.js exists with appropriate export, you dont need to mention filename in your import path, just folder name, then needed file will be found automatically